src/main/java/net/helpscout/api/HTTPMethodWrapper.java

```diff
@@ -0,0 +1,70 @@
+package net.helpscout.api;
+
+/**
+ * @Author: ivan
+ * Date: 22.08.16
+ * Time: 23:19
+ */
+public class HTTPMethodWrapper {
+
+    private final static String METHOD_GET = "GET";
+    private final static String METHOD_POST = "POST";
+    private final static String METHOD_PUT = "PUT";
+    private final static String METHOD_DELETE = "DELETE";
+
+    private String apiKey;
+    private String baseUrl;
+
+    public HTTPMethodWrapper() {
+    }
+
+    public void setApiKey(String apiKey) {
```

Comment:
You could use `@Setter` annotation on the class level and you'll get your setters generated by lombok